### ISSUE TITLE
test

### ISSUE THREAD

**Issue by AlexandrShah** [avocode/avocode#2017]
---
* **Version**: 2.26.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 328940
---
```
TypeError: Cannot read property 'l' of undefined
    at Object.rectToBounds (C:\Users\Александр\AppData\Local\avocode\app-2.26.0\resources\app.asar\node_modules\@avocode\stage\dist\utils\rendering.js:1:3087)
    at t.preview (C:\Users\Александр\AppData\Local\avocode\app-2.26.0\resources\app.asar\node_modules\@avocode\stage\dist\artboards\artboard.js:1:12044)
    at e.value (file:///C:/Users/%D0%90%D0%BB%D0%B5%D0%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4030358)
    at t.describeLayers (file:///C:/Users/%D0%90%D0%BB%D0%B5%D0%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4074801)
    at e.value (file:///C:/Users/%D0%90%D0%BB%D0%B5%D0%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80/AppData/Local/avocode/app-2.26.0/resources/app.asar/dist/avocode.min.js:1:4245268)
    at e.value (file:///C:/Users/%D0%90%D0%BB%D0%B5%D0%BA%D1%81%D0%B0%D0%BD%D0%B4%D1%80/AppData/Local/avocode/app-2.2
```

**Comment by jakubzitny** [comment]
Hi @AlexandrShah, thanks for reporting this. We can already see this bug in our tracking system. We'll try to fix it as soon as possible.

**Comment by jakubzitny** [comment]
Hi, we've just released Avocode 2.26.1, this problem should be fixed. Please update and let us know if you have any more problems 😉
